examples/interactionsBot/bot.py

```diff
@@ -29,9 +30,10 @@ async def on_ready(self):
 
 bot = BotClass()
 
-for filename in os.listdir("./cogs"):
-    if filename.endswith(".py"):
-        bot.load_extension(f"cogs.{filename[:-3]}")
+py_files = list(file.group(1) if PY_FILE_REGEX.match(file) for file in os.listdir("./cogs")) # type: ignore
+# mypy counts "if" statement inside a list comprehension as an "if/else" statement.

```

Comment:
so from a beginner standpoint, which this is mean't to target "what is mypy? what is and if/else statement?"

also this seems like you've just over complicated loading cogs for no reason?

Reply:
you're right.